search.md

```diff
@@ -9,7 +9,7 @@ Making the leap from text to coordinates is an intricate and challenging process
 All Mapzen Search requests share the same format:
 
 ```
-   https://search.mapzen.com/v1/search?text=London&api_key=mapzen-xxxxxx
+   https://search.mapzen.com/v1/search?text=London&api_key=your-mapzen-api-key
    \___/   \_______________/\__/\_____/\__________/\___________________/
```

Comment:
should we extend the last line to come to the end of the api key placeholder text for consistency?